Tests/PackageModelTests/SwiftSDKBundleTests.swift

```diff
@@ -551,4 +551,93 @@ final class SwiftSDKBundleTests: XCTestCase {
             ),
         ])
     }
+
+    func testConfigureSDKRootPath() async throws {
+        func createConfigurationStore() async throws -> (SwiftSDKConfigurationStore, FileSystem) {
+            let (fileSystem, bundles, swiftSDKsDirectory) = try generateTestFileSystem(
+                bundleArtifacts: [
+                    .init(id: testArtifactID, supportedTriples: [arm64Triple, i686Triple]),
+                ]
+            )
+            let system = ObservabilitySystem.makeForTesting()
+
+            var output = [SwiftSDKBundleStore.Output]()
+            let store = SwiftSDKBundleStore(
+                swiftSDKsDirectory: swiftSDKsDirectory,
+                hostToolchainBinDir: "/tmp",
+                fileSystem: fileSystem,
+                observabilityScope: system.topScope,
+                outputHandler: {
+                    output.append($0)
+                }
+            )
+
+            let archiver = MockArchiver()
+            for bundle in bundles {
+                try await store.install(bundlePathOrURL: bundle.path, archiver)
+            }
+
+            let hostTriple = try Triple("arm64-apple-macosx14.0")
+            let sdk = try store.selectBundle(
+                matching: testArtifactID,
+                hostTriple: hostTriple
+            )
+
+            XCTAssertEqual(sdk.targetTriple, targetTriple)
+            XCTAssertEqual(output, [
+                .installationSuccessful(
+                    bundlePathOrURL: bundles[0].path,
+                    bundleName: AbsolutePath(bundles[0].path).components.last!
+                )
+            ])
+
+            let config = try SwiftSDKConfigurationStore(
+                hostTimeTriple: hostTriple,
+                swiftSDKBundleStore: store
+            )
+
+            return (config, fileSystem)
+        }
+
+        do {
+            let (config, _) = try await createConfigurationStore()
+            let args = SwiftSDK.PathsConfiguration<String>()
+            let configSuccess = try config.configure(
+                sdkID: testArtifactID,
+                targetTriple: nil,
+                showConfiguration: false,
+                resetConfiguration: false,
+                config: args
+            )
+            XCTAssertEqual(configSuccess, false, "Expected failure for SwiftSDKConfigurationStore.configure with no updated properties")
+        }
+
+        let targetTripleConfigPath = AbsolutePath("/sdks/configuration/\(testArtifactID)_\(targetTriple.tripleString).json")
+
+        do {
+            let (config, fileSystem) = try await createConfigurationStore()
+            var args = SwiftSDK.PathsConfiguration<String>()
+            args.sdkRootPath = "/some/sdk/root/path"
+            let configSuccess = try config.configure(sdkID: testArtifactID, targetTriple: targetTriple.tripleString, showConfiguration: false, resetConfiguration: false, config: args)
```

Comment:
I suspect this line is still over the limit in terms of length. Would you mind applying SwiftFormat on this file per `CONTRIBUTING.md`?

Reply:
SwiftPM doesn't use swift-format, but SwiftFormat, could that be the reason?

Reply:
Doh! Of course. 🙄